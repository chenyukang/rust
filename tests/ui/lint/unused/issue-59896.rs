#![deny(unused_imports)]

struct S;

fn main() {
    use S; //~ ERROR redundant import

    let _s = S;
}
