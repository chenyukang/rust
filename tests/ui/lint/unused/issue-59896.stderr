error: redundant import: `S`
  --> $DIR/issue-59896.rs:6:9
   |
LL | struct S;
   | --------- the item `S` is already defined here
...
LL |     use S;
   |         ^
   |
note: the lint level is defined here
  --> $DIR/issue-59896.rs:1:9
   |
LL | #![deny(unused_imports)]
   |         ^^^^^^^^^^^^^^

error: aborting due to 1 previous error

